app/models/current_assignment.rb

```diff
@@ -9,23 +10,35 @@ def user_id
     data.fetch(:user_id)
   end
 
-  def assigned_to_user(user)
-    user.id.to_s == user_id
+  def assigned_to_user?(user)
+    assigned? && user.id.to_s == user_id
   end
 
   def assigned?
-    !latest_assignment.nil?
+    !user_id.nil?
   end
 
   private
 
   def data
-    return { user_id: nil, user_name: 'Unassigned' } unless latest_assignment
+    @data ||= projection.run(Rails.application.config.event_store).fetch(:user)
```

Comment:
Would be neat to wrap this call to event store in a cleaner method with less chaining. Like a global `application_event_store` so we done have to use the full path across the app :)

Reply:
Agree. I imagine we'll have a mixin for event sourced classes.